apps/bezier-react/src/components/KeyValueListItem/KeyValueListItem.styled.ts

```diff
@@ -1,126 +1,20 @@
-/* External dependencies */
-import { isNil } from 'lodash-es'
-
 /* Internal dependencies */
-import { css, ellipsis, styled } from 'Foundation'
-import { InterpolationProps } from 'Types/Foundation'
-import { AdditionalColorProps } from 'Types/ComponentProps'
-import { Icon } from 'Components/Icon'
-import { Tooltip } from 'Components/Tooltip'
-import { Text } from 'Components/Text'
-
-const alignRight = css`
-  margin-left: auto;
-`
-
-interface WrapperProps extends InterpolationProps {
-  multiline: boolean
-}
-
-const Wrapper = styled.div<WrapperProps>`
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-
-  &:hover {
-    background-color: ${({ foundation }) => foundation?.theme?.['bg-black-lighter']};
-  }
-
-  ${({ multiline }) => multiline && css`
-    padding-bottom: 6px;
-  `}
-
-  ${({ foundation }) => foundation?.rounding?.round6}
-
-  ${({ foundation }) => foundation?.transition.getTransitionsCSS(['background-color', 'color'])}
-
-  ${({ interpolation }) => interpolation}
-`
-
-const KeyText = styled(Text)`
-  ${ellipsis(1)}
-  color: ${({ foundation }) => foundation?.theme?.['txt-black-dark']};
-`
-
-interface KeyContentWrapperProps extends InterpolationProps {}
-
-const KeyContentWrapper = styled.div<KeyContentWrapperProps>`
-  display: flex;
-  align-items: center;
-  max-width: 118px;
-  padding-left: 8px;
+import { ellipsis, styled } from 'Foundation'
+import { KeyValueListItemWrapper, KeyValueListItemContainer } from './KeyValueListItem.common.styled'
 
-  ${({ interpolation }) => interpolation}
-`
-
-interface ValueWrapperProps extends InterpolationProps {}
-
-const ValueWrapper = styled.div<ValueWrapperProps>`
-  min-width: 0;
-  /* stylelint-disable */
-  ${KeyContentWrapper} ~ & {
-    margin-left: 8px;
-  }
-  /* stylelint-enable */
-
-  ${({ interpolation }) => interpolation}
-`
-
-const Row = styled.div`
-  display: flex;
-  align-items: center;
-  padding: 4px 6px;
-`
-
-const MultiValueRow = styled.div`
+export const Wrapper = styled(KeyValueListItemWrapper)`
   display: flex;
   align-items: center;
-  padding: 0 6px;
 `
 
-const ActionIcon = styled(Icon)``
-
-interface ActionWrapperProps extends AdditionalColorProps<['hoverBackground', 'hoverIcon']> {
-  show: boolean
-}
-
-const ActionIconTooltip = styled(Tooltip)`
-  ${alignRight}
+export const KeyItemContainer = styled(KeyValueListItemContainer)`
+  flex: 1;
+  height: 28px;
```

Comment:
```suggestion
  height: 20px;
```

<img width="414" alt="스크린샷 2022-05-24 오후 3 36 54" src="https://user-images.githubusercontent.com/25701854/169964690-f6fbcbe8-c8de-4ea6-b472-9ada280cf86c.png">
<img width="386" alt="스크린샷 2022-05-24 오후 3 37 01" src="https://user-images.githubusercontent.com/25701854/169964700-0c437a6c-4a51-4803-908d-623c53e89154.png">

Reply:
보고 있는 28px의 영역이 onClickKey, onClickValue에 해당되는 hover 영역입니다. 28px이 min으로 되어야합니다.
Wrapper에서 높이를 제한하지 않기 때문에 28px로 괜찮아보입니다.

Reply:
padding (4px 6px) 이 있어서 hover되는 영역의 실제 높이가 36px로 늘어납니다.

Reply:
![Screen Shot 2022-05-24 at 3 44 45 PM](https://user-images.githubusercontent.com/16330024/169965889-1a7e0dbb-ee67-4fd2-a131-7a5e837c693a.png)

Stack으로 바꾸면서 그렇게되나 보군요 한번 더 확인해봐야겠네요.